tests/nightly/test_np_large_array.py

```diff
@@ -335,30 +298,12 @@ def test_bincount():
     assert B[-1] == INT_OVERFLOW - 1
     B.backward()
     assert A.grad.shape == (INT_OVERFLOW, )
-    assert A.grad[0] == 0
-
-# broken
-# TODO add backward test after forward is fixed
-@use_np
-@pytest.mark.skip(reason='Does not support large tensor; to be fixed')
-def test_bitwise_family():
-    def batch_check(x1, x2, funcs):
-        for f in funcs:
-            y = f(x1, x2)
-            one = np.ones((1), dtype='int32')
-            assert y.shape == (INT_OVERFLOW, 2)
-            assert y[0][0] == f(one, one)
-    # test on broadcast input
-    A = np.ones((INT_OVERFLOW, 1), dtype='int32')
-    B = np.ones((INT_OVERFLOW, 2), dtype='int32')
-    batch_check(A, B, [np.bitwise_and, np.bitwise_or, np.bitwise_xor])
-    C = np.bitwise_not(A)
-    assert C.shape == (INT_OVERFLOW, 1)
-    assert C[0] == np.bitwise_not(np.ones((1), dtype='int32')) 
+    assert A.grad[0] == 0 
 
 @use_np
 def test_blackman():
     A = np.blackman((INT_OVERFLOW))
+    npx.waitall()
```

Comment:
What does this operator do? Why do you have to use waitall() ?

Reply:
This operator returns the blackman window of the given shape. Yes there is a formula for each term, I will change waitall() into a value check